application/backend/src/main/resources/apps/luna/components/carousel/carousel.html

```diff
@@ -0,0 +1,57 @@
+<!--/*
+    Copyright (C) 2022 Dynamic Solutions
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/-->
+
```

Comment:
I added the some style definitions here to make the slider somewhat look like a slider :). I think it's not the best place, so @paweljozwicki please comment if it needs to be moved. I was thinking about a frontend module in backend folder, but this project is missing it entirely at this point.

Using style tags here is also messing up the author experience for this component for some reason, so I used data-sly-test to not apply this styles in edit mode.

Reply:
Ok, let's leave it as it is for now.